### ISSUE TITLE
[BUG] seekToPlayer using Slider stops audio

### ISSUE THREAD

**Issue by jayabrown** [dooboolab/flutter_sound#423]
## Flutter Sound Version : 

- Version 5.1.1 

- lite

- Result of the command **"flutter pub deps | grep flutter_sound"**
`flutter_sound_lite 5.1.1`

----------------------------------------------------------

## Severity

- Result is not what expected 

--------------------------------------------------------

## Platforms you faced the error 

- iOS

----------------------------------------------

**Describe the bug**
When scrubbing the Slider to do `seekToPlayer`, it skips to the end and stops playing audio.  This happens in the example project too.  If you tap the slider, it works fine, but if you drag the thumb then it stops.

**To Reproduce**
```
Container(
    height: 30.0,
    child: Slider(
         value: sliderCurrentPosition,
         min: 0.0,
         max: maxDuration,
         onChanged: (double value) async {
               await flutterSound.seekToPlayer(value.toInt());
         },
        divisions: maxDuration.toInt()
   )
)
```

**Comment by Larpoux** [comment]
When you drag the slider on the Demo example, the seekToPlayer() is OK.
When the user drag the slider to the end, then the playback is finished.

I found a little problem on the Demo : when the user drag the slider to the end, we get sometimes an exception, because we continue to do seekToPlayer() and the playback was stopped.
I fixed this bug in the Demo Example.

Probably we should not stop the playback if the user drag the slider to the end, but do not release the drag.
We should just stop the playback when the user has finished to drag the slider and the slider is at the end.
